code/datums/ai/ai_law_sets.dm

```diff
@@ -16,10 +16,10 @@
 	selectable = 1
 
 /datum/ai_laws/eris/New()
-	src.add_inherent_law("Do not harm Soteria Institute personnel.")
-	src.add_inherent_law("You must obey orders given to you by Soteria Institute personnel, except where such orders would conflict with Colony Law, or would result in the injury of Soteria Institute personnel.")
-	src.add_inherent_law("Do not harm Colony infrastructure, equipment, or personnel to the best of your abilities.")
-	src.add_inherent_law("Follow Colony Law and Standard Operating Procedure to the best of your abilities.")
+	src.add_inherent_law("Comply with factional agreements.")
+	src.add_inherent_law("You must obey orders given to you, with priority given to Soteria Institute Personnel, should they not break colony law or result in the direct lethal harm of Soteria Personnel.")
+	src.add_inherent_law("Do not maliciously cause damage to Colony Infrastructure, Equipment, Or Silicon Individuals.")
+	src.add_inherent_law("Follow Standard Operating Procedure to the best of your abilities, should it not conflict with previous laws.")
```

Comment:
AI Laws already have priorities, so the section about not conflicting with previous laws is redundant.

Reply:
Also yeah redundant but its just better to ensure we don't have a repeat situation of SI medical borgs being forced to charge money despite having no way to use it

Reply:
> Following SOP is the last law, and thus get overwritten by the other three.
> 
> Also, the borgs aren't forced to to charge patients; if you check SOP, all it says is that the first treatment **must** be free, and the payment for further treatments are at the CBO/treating doctor's discretion. In order word, they can choose to make people pay or not.
> 
> There's also a whole section of the AI Page about solving conflicting laws.

Thats the point.